### PR TITLE
[YUNIKORN-1177] Add Chinese translation for user guide workloads documents

### PR DESCRIPTION
**Pull request by cdmikechen**
JIAR https://issues.apache.org/jira/browse/YUNIKORN-1177

Translate flow documents:
- [x] run spark
- [x] run flink
- [x] run tensorFlow

image

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/user_guide/workloads/run_flink.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## Standalone（独立）模式
+
+请关注 [Kubernetes 设置](https://ci.apache.org/projects/flink/flink-docs-stable/ops/deployment/kubernetes.html) 以获取 standalone 部署模式的细节和示例。
+在这种模式下，我们可以直接在 Deployment/Job spec 中添加需要的标签（applicationId 和 queue）来使用 YuniKorn 调度器运行 flink 应用程序，以及 [使用 YuniKorn 调度器运行 workloads](#run-workloads-with-yunikorn-scheduler) .
+
+## Native（原生）模式
+
+请关注 [原生 Kubernetes 设置](https://ci.apache.org/projects/flink/flink-docs-stable/ops/deployment/native_kubernetes.html) 以获取原生部署模式的细节和示例。
+只有 flink 1.11 或更高版本才支持在 native 模式下使用 YuniKorn 调度程序运行 flink 应用程序，我们可以利用两个 flink 配置 `kubernetes.jobmanager.labels` 和 `kubernetes.taskmanager.labels` 来设置所需的标签。
```
Comment (yuchaoran2011) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/user_guide/workloads/run_flink.md:40]: Could you change occurrences of `flink` to `Flink`?

**Review by yuchaoran2011 — CHANGES_REQUESTED** [review]
Some minor nits, otherwise looks good. Thanks for working on it

**Comment by cdmikechen** [comment]
@yuchaoran2011
Hi~ I'm sorry it took so long to fix this problem...

**Comment by yuchaoran2011** [comment]
No worries. Thanks for following up on it!

**Review by yuchaoran2011 — APPROVED** [review]

**Comment by cdmikechen** [comment]
@yuchaoran2011 
Hi~ I've seen our build is failed. I checked log and found that 
https://github.com/apache/yunikorn-site/runs/6629810151?check_suite_focus=true
image

Do we need to upgrade the docker image version?

**Comment by yuchaoran2011** [comment]
@cdmikechen Thanks for the reminder. It should be ok to make a minor version update of node. I just created at PR https://github.com/apache/yunikorn-site/pull/160

**Pull request merged**